wifimgr.py

```diff
@@ -15,6 +15,17 @@
 server_socket = None
 
 
+def _unquote(s):
```

Comment:
hmm, isn't there something in urllib (or elsewhere) for that already?

Reply:
In cPython it's possible to do that with `urllib`, but Micropython doesn't have `urllib` and apparently nothing similar.

Reply:
ok, in that case one could consider copying quote/unquote from the cpython stdlib - just to avoid issues in new code and maybe be able to use it from some lib later in case micropython adds that lib or there is some external lib on pypi (use same interface).

Reply:
I've just tried that and I don't think it would be a good idea, because the function depends on about 70 lines of code, it needs more resources and had to be modified to run on Micropython. What I think would be better to just rename the function to the `urllib` counterpart ( `unquote_plus` [docs](https://docs.python.org/2/library/urllib.html#urllib.unquote_plus) ).

Reply:
ok, if you can support the same interface and semantics, using the same name makes sense.